.github/workflows/pr-env-destroy.yml

```
@@ -83,8 +83,7 @@ jobs:
         run: |
           terraform -chdir=terraform/infrastructure destroy \
             --var-file=etc/dev.tfvars \
-            --var assume_account=${{ steps.get_account_id.outputs.aws_account_id }} \
-            --var assume_role=terraform \
+            --var assume_role_arn=${{ secrets.DEPLOY_ROLE_ARN }} \
             -auto-approve
 
```

Suggestion - Could also clean up the Terraform workspace

```suggestion
      - name: Cleanup Terraform Workspace
        run: |
          terraform workspace select default
          terraform workspace delete ${{ needs.set-environment-id.outputs.environment_id }}

```

```

---

added